### ISSUE TITLE
A11Y | 

### ISSUE THREAD

**Issue by hilcurtis** [Esri/global-nav#330]
**ID**
1843920469

**Violation**
Provide a valid label for form fields

**Module Name**
03a Header Menu - Top Row

**Description**
[Issue]
A hidden button element with collapsed state available in the header section. This button does not have a mechanism that allows an accessible name value to be calculated.

[User Impact]
When buttons do not explicitly include a label, assistive technologies may incorrectly render the label or provide no label at all to users. When labels are not present or are incorrect, users of assistive technologies may not be able to complete an action.

[Code Reference]
`<button class="esri-header-inline-title-control" id="esri-header-inline-title-control" tabindex="-1" aria-expanded="false" aria-controls="esri-header-inline-title-content"></button>`

**Note**
 [Recommendation]
Currently the purpose of this button is not clear as per the UI.
There are two scenarios for this button, the first one if it appears on the screen within certain condition it must be visual and should posses a valid programmatic label. In second case while it is hidden from all user, developer should use a display="none" property or otherwise this button should be removed from the DOM. 
Alternatively, if the button has any important function and it is purposely hidden, developer should provide an valid label to the button using either text or an aria-label.

[Compliant Code Example]
`<button class="esri-header-inline-title-control" id="esri-header-inline-title-control" tabindex="-1" aria-expanded="false" aria-controls="esri-header-inline-title-content">{use valid button text}</button>`

![image](https://user-images.githubusercontent.com/34460455/100175014-76482d80-2e82-11eb-9899-7c4bd5a1265c.png)
